src/js/playlist/playlist.js

```diff
@@ -49,6 +30,35 @@ export function validatePlaylist(playlist) {
         throw new PlayerError(MSG_CANT_PLAY_VIDEO, 630);
     }
 }
+
+export function normalizePlaylistItem(model, item, feedData) {
+    const providers = model.getProviders();
+    const preload = model.get('preload');
+    const itemFeedData = Object.assign({}, feedData);
+    delete itemFeedData.playlist;
+
+    item = Object.assign({}, item);
```

Comment:
Can we define a new const for the copied item (no-params-reassign).

`const playlistItem = Object.assign({}, item);`